dpti/equi.py

```diff
@@ -600,6 +601,35 @@ def post_task(iter_name, natoms=None, is_water=None):
     return info_dict
 
 
+def run_task(task_name, machine_file):
+    task_dir_list = [task_name]
+    work_base_dir = os.path.join(os.getcwd())
```

Comment:
Consider using pathlib's Path objects for file and directory handling. This can make the code more readable and robust. For example, `work_base_dir = os.path.join(os.getcwd())` can be replaced with `work_base_dir = Path.cwd()`. [medium]